doc/source/amazon/step-zero-aws.rst

```diff
@@ -0,0 +1,172 @@
+.. _amazon-aws:
```

Comment:
I'm assuming that you haven't added *new* content to the provider-specific sections, so not giving them as close of a look for wording etc, let me know if that's wrong and you'd like a closer look!

Reply:
Yep. No big changes to provider content except links to images from moving the file.